example-apps/callback/test/creates.js

```diff
@@ -0,0 +1,63 @@
+/* globals describe, it */
+
+require('should');
```

Comment:
we're writing new tests w/ `jest` and it's assertion functions now, see [the typescript test](https://github.com/zapier/zapier-platform/blob/79cec9d851a0123b70e66ca8ede7b9e9c2ea4521/example-apps/typescript/src/test/creates.test.ts)